inc/schema/aioseop-context.php

```diff
@@ -398,26 +422,44 @@ public static function get_object( $type, $key, $args = array() ) {
 		$object = false;
 		switch ( $type ) {
 			case 'var_site':
+			case 'var_date':
+			case 'var_date_year':
+			case 'var_date_month':
+			case 'var_date_day':
 				$object = array(
 					'context_type' => $type,
 					'context_key'  => $key,
 				);
 				break;
+
 			case 'WP_Site':
+				/*
+				 * PHP 5.2 conflict. Could merge WP_Site & WP_Post together after WP 5.1 becomes the required version.
```

Comment:
What's the actual conflict that happens here?

Reply:
It was an issue with dynamically accessing a method.

`Parse error: syntax error, unexpected T_PAAMAYIM_NEKUDOTAYIM`